frontend/src/components/QueueTable.js

```diff
@@ -58,9 +65,21 @@ function QueueTable({ queueData: { name, rows } }) {
   return (
     <Paper className={classes.root}>
       <MaterialTable
-        title={name}
+        title={queueStore.queueStats[queueData["queueData"]].name}
         className={classes.table}
-        data={rows}
+        options={{
+          search: false,
+        }}
+        data={queueStore.queues[queueData["queueData"]].map(x => ({
+          urgency: x.urgency,
+          last: x.participant.last_name,
+          uid: x.participant.pp_id,
+          timeElapsed: moment(x.status.created_at).format("LT"),
+          status: x.status.event_type,
+          service: x.service.name,
+          seen: false,
+          Notes: false,
```

Comment:
Should `Notes` be lowercase? And is it supposed to be boolean?

Reply:
Yes, fixed